src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -282,7 +282,8 @@ private static Assembly ResolveAssembly(object sender, ResolveEventArgs args)
             return null;
         }
 
-        internal DurableOrchestrationClient GetClient(OrchestrationClientAttribute attribute)
+        // To be discussed if this is an acceptable approach 
+        internal virtual DurableOrchestrationClient GetClient(OrchestrationClientAttribute attribute)

```

Comment:
I'm fine with making methods virtual. In this case should we make it `protected internal` so others can benefit from it? It feels strange to have an `internal virtual` method since it only works for unit tests that are configured for access to internal members..

Reply:
Thanks